test/config_test/example_configs_test.cc

```diff
@@ -15,7 +15,11 @@ TEST(ExampleConfigsTest, All) {
   RELEASE_ASSERT(::getcwd(cwd, PATH_MAX) != nullptr);
   RELEASE_ASSERT(::chdir(directory.c_str()) == 0);
 
-  EXPECT_EQ(26UL, ConfigTest::run(directory));
+#ifdef __APPLE__
```

Comment:
nit: Can you add a comment on which configs are different here just for future reference? We should be able to keep this up to date given that this check needs to keep happening.